### ISSUE TITLE
Implement differential learning rate across layers (pretrained embedding should have smaller learning rate)

### ISSUE THREAD

**Issue by epwalsh** [epwalsh/pytorch-crf#78]
See https://pytorch.org/docs/stable/optim.html#per-parameter-options.

TODO:

- [x] update SGD
- [x] update tests for SGD
- [x] update train.py
- [x] update other Optimizers
- [ ] test all other optimizers
